README.rst

```diff
@@ -49,6 +49,16 @@ Install Ray with: ``pip install ray``. For nightly wheels, see the
 .. _`RLlib`: https://docs.ray.io/en/latest/rllib/index.html
 .. _`ecosystem of community integrations`: https://docs.ray.io/en/latest/ray-overview/ray-libraries.html
 
+
+Why Ray?
+--------
+
+Today's ML workloads are increasingly compute intensive. As convenient as they are, single-node development environments such as your laptop cannot scale to it: there is a need for a unified way to scale from a laptop to a cluster.
+
+Ray is this unified way to scale Python and AI applications.
+
+With Ray, you can seamlessly scale the same code from laptops to clusters. Ray's underlying generality also means it can handle any workload with the best performance possible-- so you can run one system instead of many.
```

Comment:
```suggestion
Today's ML workloads are increasingly compute-intensive. As convenient as they are, single-node development environments such as your laptop cannot scale to meet these demands.

Ray is a unified way to scale Python and AI applications from a laptop to a cluster.

With Ray, you can seamlessly scale the same Python code from a laptop to a cluster. Ray is also designed to be general-purpose, meaning that it can handle any workload and still provide the best performance. If your application is written in Python, you can scale it with Ray. No other systems required.
```

Not really sure about the "best performance" part...